From e86a0693d494d54fb9ff8a68bc2b9c6d6fd4d30a Mon Sep 17 00:00:00 2001
From: Will Thompson <will@willthompson.co.uk>
Date: Mon, 17 Apr 2023 12:23:25 +0100
Subject: [PATCH 2/2] gtk: Add translation info to AppStream metainfo

This allows information about per-language translation percentages to be
extracted at build-time and included in the repository's appstream file.
Software centres such as GNOME Software use this to indicate to the user
whether the app is translated into their language.

https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-translation
---
 gtk/transmission-gtk.metainfo.xml.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gtk/transmission-gtk.metainfo.xml.in b/gtk/transmission-gtk.metainfo.xml.in
index 15a6656b9..2d9f00192 100644
--- a/gtk/transmission-gtk.metainfo.xml.in
+++ b/gtk/transmission-gtk.metainfo.xml.in
@@ -53,4 +53,5 @@ Copyright 2017 Endless Mobile, Inc.
     <release date="2023-02-08" version="4.0.0" />
     <release date="2020-05-03" version="3.00" />
   </releases>
+  <translation type="gettext">transmission-gtk</translation>
 </component>
-- 
2.30.2

